test/dlc_tests/container_tests/bin/gdrcopy/test_gdrcopy.sh

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+#!/usr/bin/env bash
+sanity | grep 'Failures: 0, Errors: 0' &> /dev/null
+if [ $? == 1 ]; then
```

Comment:
Could you please change this to `!= 0` instead, so that any error causes this test to fail, rather than only an exit code of `1`?